README.md

```diff
@@ -120,6 +120,7 @@ type Props = {
     pingOnlyIfOffline?: boolean = false,
     pingInBackground?: boolean = false,
     httpMethod?: HTTPMethod = 'HEAD',
+    customHeaders?: {} = {},
```

Comment:
Type should be:

```js
type HTTPHeaders = {
	[key: string]: string
}
...

customHeaders?: HTTPHeaders = {}
```

You can add the new type [here](https://github.com/rgommezz/react-native-offline/blob/master/src/types.ts) and export it accordingly.